src/js/client/apply.js

```diff
@@ -1,10 +1,11 @@
 const $ = require('jquery');
 const { createApplicationForm } = require('js/shared/application-form');
+const { createTeamForm } = require('js/shared/team-form');
 const renderForm = require('js/shared/render-form');
 
 /**
  * jQuery's .serializeArray does not give us file input values. While we can't get useful
- * information about files' content easily, we can get it's name for validation easily.
+ * information about files' content easily, we can get its name for validation easily.
```

Comment:
We _need_ this. These kinds of mistakes are embarrassing when they're published.